solidity/contracts/Coordinator.sol

```diff
@@ -348,19 +348,13 @@ contract Coordinator is ChainlinkRequestInterface, CoordinatorInterface {
     _;
   }
 
-  modifier hasAvailableFunds(uint256 _amount) {
-    require(withdrawableTokens[msg.sender] >= _amount, "Amount requested is greater than withdrawable balance");
-    _;
-  }
-
   /**
    * @dev Allows the oracle operator to withdraw their LINK
    * @param _recipient is the address the funds will be sent to
    * @param _amount is the amount of LINK transfered from the Coordinator contract
    */
   function withdraw(address _recipient, uint256 _amount)
     external
-    hasAvailableFunds(_amount)
```

Comment:
My instinct would be to leave this check in. It's additional defense against a potentially catastrophic error, it explicitly registers the intent whereas `SafeMath#sub` is implicit, and it's cheap in terms of gas, complexity and readability.